db/migrate/20250414121956_migrate_ovirt_resources.rb

```diff
@@ -0,0 +1,21 @@
+class MigrateOvirtResources < ActiveRecord::Migration[7.0]
+  # Follow up to the ovirt migration task (lib/tasks/ovirt.rake).
+  def up
+    compute_resources = ComputeResource.unscoped.where(type: 'Foreman::Model::Ovirt')
+
+    compute_resources.each do |cr|
+      Rails.logger.debug "Processing data for '#{cr.name}' compute resource."
+
+      cr.hosts.each do |host|
+        Rails.logger.debug "Disassociating host '#{host.name}' from the compute resource."
+        host.disassociate!
+      end
+
+      Rails.logger.debug "Removing compute resources and compute profiles from host groups."
+      cr.hostgroups.update_all(compute_resource_id: nil, compute_profile_id: nil)
+
+      Rails.logger.debug "Removing '#{cr.name}' compute resource."
+      cr.destroy
+    end
+  end
```

Comment:
Would you consider adding a `down` method to this migration? Since it performs destructive actions like deleting compute resources, it would be helpful to explicitly mark it as irreversible by raising `ActiveRecord::IrreversibleMigration`. This makes the intent clearer and avoids confusion if someone tries to roll back the migration later.

```
  def down
    raise ActiveRecord::IrreversibleMigration, "Cannot restore deleted compute resources"
  end
```